docs/UserGuide.md

```diff
@@ -321,11 +321,15 @@ Displays help for all/certain commands.
 
 Format: (All command) `help` or (Certain command) `help COMMAND`
 
+Showing all commands:
 ![help](images/screenshots/help.png)
 
+Showing one specific command:
+![help](images/screenshots/help2.png)
+
 Examples : 
 * `help` (Shows all command and its format)
-* `help tag` (shows tag command description, format and examples usage)
+* `help tag` (shows cd command description, format and examples usage)
```

Comment:
Shouldn't this still be showing help for tag?

Reply:
The updated image is cd, so I changed it to cd

Reply:
emmm, just found did not change the help tag to help cd

Reply:
thank you for pointing that out